customization/dark-theme.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: doc
+title: Dark Theme in Dom0 and DomU
+permalink: /doc/dark-theme/
+---
+
+Dark Theme in Dom0
+==================
+
+Dark KDE in Dom0
+----------------
+
+The following text describes how to change the default light theme to a dark theme. This is just an example, feel free to adjust the appearance to your taste.
+
+The image below shows the default light theme after installation.
+![begin light theme](/attachment/wiki/Dark-Theme/kde-fresh-installed-standard.png)
+
+This is the result after applying the steps described here.
+![end result dark theme](/attachment/wiki/Dark-Theme/kde-end-result.png)
+
+1 - Change `Workspace Appearance`
+
+  1. Open the `Workspace Appearance` window
+
+    ~~~
+    Qubes Menu -> System Tools -> System Settings -> Workspace Appearance
+    ~~~
+
+   ![Workspace Appearance](/attachment/wiki/Dark-Theme/kde-app-appearance-menu-style.png)
+
+  2. Go to `Desktop Theme`
+
+   ![Desktop Menu](/attachment/wiki/Dark-Theme/kde-appearance-settings-desktop-theme-oxygen.png)
+
+  3. Select `Oxygen` and `Apply` the change
+
+2 - (Optional) Remove blue glowing task items
+
+![blue glowing task bar items](/attachment/wiki/Dark-Theme/kde-taskbar-blue-glowing-border.png)
+
+  1. Adjust Oxygen `Details`
+
+    ~~~
+    Qubes Menu -> System Tools -> System Settings -> Workspace Appearance -> Desktop Theme -> Details (Tab)
+    ~~~
+
+  2. Select `Oxygen`
+
+  3. Change `Theme Item -> Task Items` from `Oxygen Task Items` to `Air Task Items`
+
+   ![Change Task items look](/attachment/wiki/Dark-Theme/kde-desktop-theme-details.png)
+  4. Apply changes
+
+    ![task bar items blue glowing removed](/attachment/wiki/Dark-Theme/kde-taskbar-blue-glowing-removed.png)
+
+3 - Change `Application Appearance`
+
+  1. Open the `Application Appearance` window
```

Comment:
Fixing the numbering here will fix the image embedding. (Currently, the images do not show up in this section.)